javaparser-testing/src/test/resources/com/github/javaparser/bdd/visitor_scenarios.story

```diff
@@ -73,4 +73,4 @@ Given a CompilationUnit
 Given a VoidVisitorAdapter with a visit method that asserts sensible line positions
 When the "JavaConcepts.java" is parsed
 When the CompilationUnit is visited by the PositionTestVisitor
-Then the total number of nodes visited is 1425
+Then the total number of nodes visited is 1427
```

Comment:
This test is a bit annoying to change every time. Is it still a good idea to have it?